README.md

```diff
@@ -49,6 +49,21 @@ To use mvn test, please first run an opengemini server locally. We recommend usi
 docker run -p 8086:8086 --name opengemini --rm opengeminidb/opengemini-server
 ```
 
+### Create Client
+
+```java
+import io.opengemini.client.jdk.OpenGeminiJdkClient;
```

Comment:
please add all the using class, such as `import io.opengemini.client.jdk.Configuration;`